sopel/module.py

```diff
@@ -78,9 +78,23 @@
 
 
 def unblockable(function):
-    """Decorator which exempts the function from nickname and hostname blocking.
+    """Decorator to exempt ``function`` from nickname and hostname blocking.
+
+    This can be used to ensure events such as ``JOIN`` are always recorded::
+
+        from sopel import module
+
+        @module.event('JOIN')
+        @module.unblockable
+        def on_join_callable(bot, trigger):
+            # do something when a user JOIN a channel
+            # a blocked nickname or hostname *will* trigger this
+            pass
+
+    .. seealso::
+
+        Sopel's :meth:`~sopel.bot.dispatch` method.
```

Comment:
Shouldn't this also (or instead?) cross-reference `get_triggered_callables`? That's where `unblockable` is checked/enforced now.

And I think it needs to reference `~sopel.bot.Sopel.methodname` (but haven't test-built Sphinx yet).

Reply:
Yup, you're right, twice.